bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableTableAdminClientWrapper.java

```diff
@@ -242,6 +257,20 @@ public void dropRowRangeAsync(){
     verify(mockAdminClient).dropRowRangeAsync(request);
   }
 
+  @Test
+  public void dropRowRangeAsyncForTruncate(){
+    DropRowRangeRequest request = DropRowRangeRequest.newBuilder()
+        .setName(TABLE_NAME)
+        .setDeleteAllDataFromTable(true)
+        .build();
+
+    when(mockAdminClient.dropRowRangeAsync(request))
+        .thenReturn(immediateFuture(Empty.newBuilder().build()));
+    clientWrapper.dropRowRangeAsync(TABLE_ID, request.getRowKeyPrefix().toStringUtf8());
```

Comment:
Shouldn't this be `clientWrapper.dropRowRangeAsync(TABLE_ID, null);`?

Reply:
yes, Thanks for pointing it out. Have updated the same.